ci/requirements.yml

```diff
@@ -10,6 +10,8 @@ dependencies:
   - configobj
   - xarray
   - bottleneck
+  - matplotlib
+  - warnings
```

Comment:
`warnings` is a python builtin, you don't need to install it. This is currently causing your test failure.